ansible/roles/xl_auth/tasks/main.yml

```diff
@@ -3,15 +3,26 @@
 - name: read data for creating xl_auth secret
   shell: "cat /etc/pki/tls/private/{{ inventory_hostname }}.key"
   changed_when: no
-  register: ssh_key_contents
+  ignore_errors: yes
```

Comment:
Why `ignore_errors`?

Reply:
It's a workaround so the playbook will proceed and finish in prod-like fashion when running `vagrant up` or deploying to some temporary QA server.

Reply:
Check.